website/src/components/Install/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react'
+
+import styles from './install.module.scss'
+
+const installText = `
+docker run
+--publish 7080:7080 --publish 127.0.0.1:3370:3370 --rm
+--volume ~/.sourcegraph/config:/etc/sourcegraph
+--volume ~/.sourcegraph/data:/var/opt/sourcegraph
+sourcegraph/server:3.37.0
+`
+
+export const Install: React.FunctionComponent = () => {
+
+    const [copied, setCopied] = useState(false)
```

Comment:
Since there is no source of truth yet for a UI library/DLS, I think we should wait before implementing new features that haven't been requested. Who knows what the future holds. 🤷🏻‍♂️    Leaving this for now.